Zappy_IA_Report/DTC/Video/Tutorial3.py

```
@@ -0,0 +1,17 @@
+import pandas as pd
+
+d = {'sales': [100000, 222000, 1000000, 522000, 111111],
+    'city': ['Tampa', 'Tampa', 'Orlando', 'Jacksonville', 'Miami'],
+    'size': ['Small', 'Medium', 'Large', 'Large', 'Small']}
+
+df = pd.DataFrame(data=d)
+print(df.head())
+print(df['city'].unique())
+
+from sklearn.preprocessing import OneHotEncoder
```

Ruff (0.11.9)

Module level import not at top of file

(E402)